cmd/kev/cmd/dev.go

```diff
@@ -167,9 +167,21 @@ func runDevCmd(cmd *cobra.Command, args []string) error {
 		return err
```

Comment:
To keep err handling consistent with other cmds, just use `return displayError(err)`.

Should prob apply to all `return err` in this file.

Reply:
updated